src/functions.js

```diff
@@ -297,15 +297,24 @@ defineFunction([
 
 // No limits, symbols
 defineFunction([
-    "\\int", "\\iint", "\\iiint", "\\oint",
+    "\\int", "\\iint", "\\iiint", "\\oint", "∫", "\u222c", "\u222d", "\u222e",
 ], {
     numArgs: 0,
 }, function(context) {
+    let fName = context.funcName;
+    if (fName.length === 1) {
+        fName = {
+            "∫": "\\int",
+            "\u222c": "\\iint",
+            "\u222d": "\\iiint",
+            "\u222e": "\\oint",
+        }[fName];
+    }
```

Comment:
This feels a little weird, to have the constant in the code there.  It's a little inefficient if this gets run a lot (though perhaps slightly faster as is if it never gets used...).  Thoughts on moving it to a `const` outside the function?

Reply:
Sure. I'll move it to a constant outside the function.